synfig-studio/src/gui/dialogs/about.cpp

```diff
@@ -312,8 +314,24 @@ About::About()
 	#endif
 
 	extra_info += "\n";
-
-	extra_info += etl::strprintf(_("Built on %s" /* at %s */ "\n\n"), __DATE__ /* , __TIME__ */ );
+	{
+		const int max_date_length = 50;
```

Comment:
Maybe extract this part to `synfig/general.h`?

Reply:
I don't know.
One is for synfig/libsynfig. Other is for synfigstudio.
I don't really know if build date is really necessary to be shown to user. :)